docs/development/writingzeppelinapplication.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: page
+title: "Writing Zeppelin Application"
+description: ""
+group: development
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+### What is Zeppelin Application (Experimental)
```

Comment:
@Leemoonsoo How about changing this  title to "What is Apache Zeppelin Application? (Experimental)" with using `<h1>` instead of `<h3>`?